taxcalc/consumption.json

```diff
@@ -0,0 +1,41 @@
+{
+    "_MPC_e17500": {
+        "start_year": 2013,
+        "long_name": "Marginal propensity to consume medical expenses",
+        "description": "Defined as dollar change in medical-expense consumption divided by dollar change in income.  Must be in [0,1] range.",
```

Comment:
On Wed, 29 Jun 2016, Matt Jensen wrote:

> 
> In taxcalc/consumption.json:
> 
> > @@ -0,0 +1,41 @@
> > +{
> > +    "_MPC_e17500": {
> > +        "start_year": 2013,
> > +        "long_name": "Marginal propensity to consume medical expenses",
> > +        "description": "Defined as dollar change in medical-expense consumption div
> ided by dollar change in income.  Must be in [0,1] range.",
> 
> @martinholmer, why is it important to constrain the MPCs to [0,1]? My intuition is
> that some MPCs could be negative (are these inferior goods?) or in some cases they
> could be greater than 1 (perhaps an addictive drug that causes you to draw down
> savings once you get started)? The MPC's for these items probably are in [0,1], but
> why constrain it?

I agree, but there will be people entering percents instead of fractions 
who would be helped by this restriction.

dan

> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the
> thread.[AHvQVUEjAICJiW9n8QSdm6BFBEOiwQKiks5qQmWrgaJpZM4I0zqy.gif]
> 
> 
>